src/removeInternalOas.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from "path";
+import fs from "fs-extra";
+import { PRODUCTION_API_PATH } from "./lib/config";
+
+/**
+ * Recursively removes all files ending in '-internal.yaml' from a directory and its subdirectories
+ * @param directoryPath - The path to the directory to process
+ */
+export function removeInternalOas(directoryPath: string): void {
```

Comment:
Just curious, do we need to run this more than once? Or is this just a helper function for cleaning up the -internal files current state of the repo?

Reply:
We'll have to run it everytime we pull down the APIs as the internal OAS file will always be a part of anypoint exchange asset (at least for now)